src/core/js/drawer.js

```diff
@@ -18,17 +18,12 @@ define([
         Adapt.trigger('drawer:triggerCustomView', view, hasBackButton);
     };
 
-    var init = function() {
-        var drawerView = new DrawerView({collection: DrawerCollection});
-
-        Adapt.on('app:languageChanged', function() {
-            drawerView.remove();
-            drawerView = new DrawerView({collection: DrawerCollection});
-        });
-    };
+    Adapt.on('adapt:start', function() {
```

Comment:
Would you mind switching to object syntax instead of the two `.on()` calls?
```js
Adapt.on({
    'adapt:start': function() { new DrawerView({ collection: DrawerCollection }); },
    'app:languageChanged': function() { Adapt.trigger('drawer:remove'); }
});
```